### ISSUE TITLE
Swapchain does not expose color_space

### ISSUE THREAD

**Issue by TJYSunset** [vulkano-rs/vulkano#1801]
Probably only a minor annoyance, but it is somewhat inconsistent.

Speaking of which, the "new" (compared to the last time I tried vulkano) triangle example does not deal with color space at all. Which is probably fine since the builder has a sane default of sRGB, but isn't the example supposed to show off all the options? (inferring from the even less used `CompositeAlpha` being in the example)

**Comment by Rua** [comment]
By "expose" do you mean a method to retrieve it from the built `Swapchain` object?

As for the examples, I don't know much about how colourspaces would be used. If you know of some way to introduce it in the triangle example, can you make a PR?

**Comment by TJYSunset** [comment]
> By "expose" do you mean a method to retrieve it from the built Swapchain object?

Yes. A getter like all the other fields have.

> can you make a PR?

I personally find the examples hard to follow (in the triangle example, everything is smashed together with walls of comments) and does not help much with writing a practical renderer (the deferred example does not even explain render passes at all and have confusing abstractions... like what exactly is a "triangle draw system"??). 

If you are OK with this, I want to do something more ambitious - create a better example. But in the mean time I need to finish my toy renderer first to make sure I get the basics right (which might take a few days).

**Comment by Rua** [comment]
I don't speak for the Vulkano project, but I think they would welcome more elaborate and less trivial examples. Something that shows not only how to write Vulkano code, but also how to make real-life use of Vulkan's features.

**Comment by Eliah-Lakhin** [comment]
The more examples we have the better! Complex examples as well. So this is definitely very welcome!

Currently we have [one example](https://github.com/vulkano-rs/vulkano/tree/master/examples/src/bin/interactive_fractal) from @hakolao that covers multiple features together.

**Comment by Eliah-Lakhin** [comment]
> I don't speak for the Vulkano project

@Rua Well, Rua, I would say that you practically already performing a primary maintainer role in this project. Not only as a regular contributor into a core of the project, but also as a person how helps with code reviews and user support. I would love to see you a part of the formal maintainer team of course if it would be interesting for you.
